tests/DCPS/Messenger/run_test.pl

```diff
@@ -29,6 +29,19 @@
 my $is_rtps_disc = 0;
 my $DCPSREPO;
 
+my $filename = "subscriber";
```

Comment:
Could this be changed in the common modules ($test->process or $test->start_process)?  It should apply to any test, and the common modules already know about file names extensions, etc.

Reply:
I could be wrong, but it looks like common module functions like 'process' do not care about extension, as they can start the process without it. I can add this to the 'process' function:

  my $extension = ".exe";
  my $executable_ext = $executable + $extension;
  if (!(-e $executable) && !(-e $executable_ext)) {
    print STDERR "ERROR: executable \"$executable\" does not exist \n";
    $self->{status} = -1;
    return;
  }

Also, messenger script does not use 'process', or 'start_process', but 'PerlDDS::create_process'. I can either change that to use process / start_process or add similar check in PerlDDS::create_process

Reply:
Some older scripts don't use "PerlDDS::TestFramework", you can migrate them to use it.

Reply:
Question: If I add the code above to 'process' sub, how would it stop the run_test script that called it, if 'process' fails? It sets $self->{status} = -1;, but I am not sure it does anything. I can start returning $self->{status}, and then check it in run_test.pl. Hope it won't break any existing scripts...

Reply:
I have added the code above to 'process' sub.